Sources/Stencil/StringsContext.swift

```diff
@@ -52,7 +52,7 @@ extension StringsFileParser {
       usingMapper: entryToStringMapper
     )
     let tables: [[String: Any]] = [[
-      "name": tableName,
+      "name": "Localizable",
```

Comment:
That present #42 will have to be merged before #41 I guess, to avoid this hard-coded value here to override the `self.tables.map { … }` in #41 right?